ampeg/src/test/java/AccountLoginTests.java

```diff
@@ -0,0 +1,16 @@
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.Test;
+
+public class AccountLoginTests extends AmpegTests {
+	@Test
+	public void ensureLoginIsSuccessful() {
+		var expectedUrl = "https://ampeg.com/account/";
+
+		var actualUrl = new HomePage(this.driver).clickSignInButton().login("miso9", "40nlyM32kn0!")
```

Comment:
Move the user name and password parameters to variables. This allows your code to be self documenting.